src/lib/components/Count/Count.js

```diff
@@ -6,12 +6,12 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Label } from 'semantic-ui-react';
+import React, { Component, useContext } from 'react';
 import Overridable from 'react-overridable';
+import { Label } from 'semantic-ui-react';
+import { AppContext } from '../';
```

Comment:
This was causing a build failure.
```suggestion
import { AppContext } from '../AppContext';
```

Reply:
I agree but thank you for working on it